src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -254,6 +254,7 @@ static void TestMain()
 Flushing
 1
 True
+True
```

Comment:
We've been trying to limit code coverage instrumentation to actual source code, so I would prefer the original behavior, but I'm certainly open to discussing this.